cmd/monaco/deploy/deploy.go

```diff
@@ -191,14 +191,26 @@ func sortConfigs(projects []project.Project, environmentNames []string) (project
 }
 
 func logProjectsInfo(projects []project.Project) {
-	log.Info("Projects to be deployed:")
+	log.Info("Projects to be deployed (%d):", len(projects))
 	for _, p := range projects {
 		log.Info("  - %s", p)
 	}
+
+	if log.DebugEnabled() {
```

Comment:
`Run Static Code Analysis / lint (pull_request) Successful in 1m`
How can it be successful, if here are obvious errors 🤔 

SA9003: empty branch (staticcheck)

Reply:
....

Reply:
fixed the issue.

I think the reporting error is an problem of the review dog action we use, which wraps the linter and posts on the PR - but it looks like this one runs "successfully" regardless of the linter's status